exchanges/bittrex/bittrex_types.go

```diff
@@ -57,30 +57,17 @@ type BulkCancelResultData struct {
 	Result     OrderData `json:"result"`
 }
 
-// NewOrder holds response data for CancelOrder
-type NewOrder struct {
-	MarketSymbol  string  `json:"marketSymbol"`
```

Comment:
// NewOrder holds response data for CancelOrder
 type NewOrder struct {
 	MarketSymbol  string  `json:"marketSymbol"`
 	Direction     string  `json:"direction"`
 	Type          string  `json:"type"`
 	Quantity      float64 `json:"quantity,string"`
 	Ceiling       float64 `json:"ceiling,string"`
 	Limit         float64 `json:"limit,string"`
 	TimeInForce   string  `json:"timeInForce"`
 	ClientOrderID string  `json:"clientOrderId"`
 	UseAwards     bool    `json:"useAwards"`
 }